nym-api/src/node_status_api/cache/mod.rs

```diff
@@ -41,12 +41,13 @@ pub struct NodeStatusCache {
 
 impl NodeStatusCache {
     /// Creates a new cache with no data.
-    fn new() -> NodeStatusCache {
+    pub(crate) fn new() -> NodeStatusCache {
         NodeStatusCache {
             inner: Arc::new(RwLock::new(NodeStatusCacheData::new())),
         }
     }
 
+    #[deprecated(note = "TODO dz: obsolete because it's used for Rocket")]
```

Comment:
the same comment I had with the storage, just remove it, because it doesn't do much apart from adding it to the managed states